backend/dataall/modules/metadata_forms/db/enums.py

```diff
@@ -0,0 +1,47 @@
+from dataall.base.api.constants import GraphQLEnumMapper
+
+
+class MetadataFormVisibility(GraphQLEnumMapper):
+    Team = 'Team Only'
+    Environment = 'Environment-Wide'
+    Organization = 'Organization-Wide'
```

Comment:
nit: isn't `Team` `Environment` `Organization` enough?

Reply:
IDK, '*-wide' was used in requirements, but I think it's not very important